src/cascadia/TerminalApp/TerminalPage.h

```diff
@@ -4,7 +4,8 @@
 #pragma once
 
 #include "TerminalPage.g.h"
-#include "Tab.h"
+#include "TerminalTab.h"
+#include "SettingsTab.h"
```

Comment:
wait... what do we need `SettingsTab.h` for? It looks like we're only working with `TerminalTab` below.